Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift

```diff
@@ -87,6 +87,7 @@ struct BasicAppSettingsView: View {
                                         forKey: KeychainKey.ownerPrivateKey.rawValue)
                                     AppSettings.setSigningKeyAddress(nil)
                                     App.shared.snackbar.show(message: "Owner key removed from this app")
+                                    Tracker.shared.setUserProperty("\(SafeTransactionSigner.numberOfKeysImported())", for: TrackingUserProperty.numKeysImported)
```

Comment:
we could use here Tracker.shared.setUserProperty("0", for: TrackingUserProperty.numKeysImported)